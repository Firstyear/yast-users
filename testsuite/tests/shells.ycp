/**
 * File:
 *  shells.ycp
 *
 * Module:
 *  Users
 *
 * Summary:
 *  Available shells listing test.
 *
 * Authors:
 *  Petr Blahos <pblahos@suse.cz>
 *
 * $Id$
 *
 */

{

    include "testsuite.ycp";
    import "Users";

    map READ = $[ "etc" : $["shells" : ["/bin/ash", "/bin/bash", "/bin/bash1", "/bin/csh", "/bin/false", "/bin/ksh", "/bin/sh"]]];

    TEST(``(Users::GetAllShells (`items)), [READ, $[], $[]], 0);
    DUMP ("==========================================================");
    TEST(``(Users::GetAllShells (`stringlist)), [READ, $[], $[]], 0);

}
