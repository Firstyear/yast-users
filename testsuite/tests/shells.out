Read	.target.size "/etc/install.inf" -1
Read	.target.tmpdir "/tmp/"
Read	.target.tmpdir "/tmp/"
Execute	.target.bash "/usr/lib/YaST2/bin/version" 0
Read	.sysconfig.language.RC_LANG ""
Read	.sysconfig.language.DEFAULT_LANGUAGE ""
Read	.target.yast2 "lang2iso.ycp" nil
Read	.target.size "/etc/sysconfig/language" -1
Read	.sysconfig.console.CONSOLE_FONT ""
Read	.sysconfig.console.CONSOLE_SCREENMAP ""
Read	.sysconfig.console.CONSOLE_UNICODEMAP ""
Read	.sysconfig.console.CONSOLE_MAGIC ""
Read	.sysconfig.console.CONSOLE_ENCODING ""
Read	.etc.shells ["/bin/ash", "/bin/bash", "/bin/bash1", "/bin/csh", "/bin/false", "/bin/ksh", "/bin/sh"]
Read	.target.size "/bin/ash" 0
Read	.target.size "/bin/bash" 0
Read	.target.size "/bin/csh" 0
Read	.target.size "/bin/false" 0
Read	.target.size "/bin/ksh" 0
Read	.target.size "/bin/sh" 0
Return	[`item (`id ("/bin/ash"), "/bin/ash"), `item (`id ("/bin/bash"), "/bin/bash"), `item (`id ("/bin/csh"), "/bin/csh"), `item (`id ("/bin/false"), "/bin/false"), `item (`id ("/bin/ksh"), "/bin/ksh"), `item (`id ("/bin/sh"), "/bin/sh")]
Dump	==========================================================
Read	.etc.shells ["/bin/ash", "/bin/bash", "/bin/bash1", "/bin/csh", "/bin/false", "/bin/ksh", "/bin/sh"]
Read	.target.size "/bin/ash" 0
Read	.target.size "/bin/bash" 0
Read	.target.size "/bin/csh" 0
Read	.target.size "/bin/false" 0
Read	.target.size "/bin/ksh" 0
Read	.target.size "/bin/sh" 0
Return	["/bin/ash", "/bin/bash", "/bin/csh", "/bin/false", "/bin/ksh", "/bin/sh"]
